src/routes/controllers/search.controller.js

```diff
@@ -0,0 +1,38 @@
+import { errorResponse, successResponse } from '../../utils/helpers.utils';
+import {
+  articleFilter,
+  authorsFilter,
+  tagsFilter,
+} from '../../utils/database.utils';
+
+/**
+ * perform custom search sitewide
+ *
+ * @param {Object} req Express request
+ * @param {Object} res Express object
+ * @returns {Array} articles, authors and tags
+ */
+export default async function customSearch(req, res) {
+  const { keyword } = req.query;
+
+  if (!keyword) {
+    return errorResponse(res, 400, 'Please input a search parameter');
+  }
+
+  try {
+    const articles = await articleFilter(keyword);
+
```

Comment:
we remove these line spaces in between variable definitions.

Reply:
I have fixed this